data/json/terrain.json

```diff
@@ -402,6 +422,7 @@
         "type" : "terrain",
         "id" : "t_pit",
         "name": "pit",
+        "description": "A steep hole that could seriously injure something if it fell in, potentially fatal if it was filled with sharp and dangerous things.  Deep enough for more advanced construction projects, and possibly to reach groundwater if constructed properly.",
```

Comment:
`fatally`

Reply:
>potentially fatally

is not grammatically correct.

Reply:
Consider:

> This story is potentially fatally flawed

> A dangerous and potentially fatally poisonous mushroom 

So, given this sentence:

> A steep hole that could seriously injure something if it fell in, potentially fatal if it was filled with sharp and dangerous things.

Let's simplify the conjunction to see what we're dealing with:

> A steep hole that could injure something fatal if it was filled with sharp and dangerous things.

See the issue?

Here's a proper version:

> A steep hole that could seriously injure something if it fell in, potentially fatally if it was filled with sharp and dangerous things.

or,

> A steep hole that could seriously injure something if it fell in – potentially fatally, if someone filled the hole with sharp and dangerous things.